### ISSUE TITLE
renewed certificate not installed into /etc/nginx

### ISSUE THREAD

**Issue by garyhooper** [Neilpang/acme.sh#1942]
I have 3 domains running on nginx. Two are fine, but one fails to install the updated certificate files upon renewal.

Examining `~/.acme.sh/domain` shows that the cert files were indeed updated. However, `/etc/nginx/certs/domain`, where they are to be installed upon renewal, remains unchanged, containing the old cert files.

The contents of `~/.acme.sh/domain/domain.conf` also shows the correct date for the updated cert, and the correct installation paths.  (They were initially installed correctly too!)

When I first encountered this, I re-executed the install command.  A new cert was not generated (not yet expired), and the existing cert files were correctly updated into `/etc/nginx/certs/domain`. Therefore, this problem is repeatable.

Why would the installation of the renewed cert files fail, and fail for only one of three domains?  (Permissions are the same for all domains in `/etc/nginx/certs` and `/etc/nginx/certs/domain`, using `755`.)

Any suggestions would be appreciated.

**Comment by FernandoMiguel** [comment]
Their renewal dates might be off sync?
Try once and only once a - -force so all of them as the same date. 
Make sure nginx is reloaded afterwards (should be part of the install hook)

**Comment by garyhooper** [comment]
@Neilpang, the first log indicated that all certs are up to date, so were skipped.

Running the second one, I see no errors.  It ends with:
> [Sun Dec  2 11:32:58 PST 2018] The intermediate CA cert is in  /home/ubuntu/.acme.sh/gluco.zone/ca.cer 
> [Sun Dec  2 11:32:58 PST 2018] And the full chain certs is there:  /home/ubuntu/.acme.sh/gluco.zone/fullchain.cer 
> [Sun Dec  2 11:32:58 PST 2018] Installing cert to:/etc/nginx/certs/gluco.zone/gluco.zone.cer
> [Sun Dec  2 11:32:58 PST 2018] Installing key to:/etc/nginx/certs/gluco.zone/gluco.zone.key
> [Sun Dec  2 11:32:58 PST 2018] Installing full chain to:/etc/nginx/certs/gluco.zone/fullchain.cer
> [Sun Dec  2 11:32:58 PST 2018] Run reload cmd: sudo systemctl reload nginx.service
> [Sun Dec  2 11:32:58 PST 2018] Reload success
> [Sun Dec  2 11:32:58 PST 2018] _on_issue_success
> [Sun Dec  2 11:32:58 PST 2018] 'dns_cf' does not contain 'dns'

Note that `'dns_cf' does not contain 'dns'` also occurs for the domains that work.

**Comment by garyhooper** [comment]
@FernandoMiguel the issue is not related to restarting nginx. The problem is that the updated cert files are not being copied from ~/.acme.sh to /etc/nginx.

**Comment by neilpang** [comment]
Closing as stale: no activity for over 18 months. If this still applies on the current release, please open a new issue with the `--debug 2` log.
